bin/server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const app = express()
+
+const initialize = () => {
```

Comment:
this function does not "initialize" express, its listening to a port and doing everything